LogMonitor/src/LogMonitor/SystemInfo.cpp

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+#include "pch.h"
+#include "SystemInfo.h"
+
+bool SystemInfo::IsAmd64() {
```

Comment:
Perhaps the system info could just be one `struct`, key/value pair kind of thing?